libs/ui/src/modals/screens/d-app-sign-confirmation/d-app-sign-confirmation.tsx

```diff
@@ -52,7 +72,7 @@ export const DAppSignConfirmation: FC = () => {
       <DAppHeader favicon={params.dAppInfo.favicon} origin={params.dAppInfo.origin} />
       <View style={styles.messageBlock}>
         <Text style={styles.mainText}>Message to sign</Text>
-        <Text style={styles.text}>{prepareSignData(messageToSign)}</Text>
+        <Text style={styles.text}>{prepareSignData(messageToSign(params.method))}</Text>
```

Comment:
`prepareSignData(messageToSign(params.method))` duplication.  
create variable and reuse it